meshio/ply/_ply.py

```diff
@@ -35,6 +36,24 @@
 numpy_to_ply_dtype = {numpy.dtype(v): k for k, v in ply_to_numpy_dtype.items()}
 
 
+cell_type_to_count = {
+    "line": 2,
+    "triangle": 3,
+    "quad": 4,
+    "polygon5": 5,
+    "polygon6": 6,
+    "quad8": 8,
```

Comment:
`quad8` should also be removed.

Reply:
It seems that PLY supports and arbitrary number of nodes in a facet. We can try and support that. line, triangle, quad map to 2, 3, 4 (how about vertex for 1?), everything above to `polygonN` (where `N` is the respective number).

Reply:
Sure, I'll do that.